src/pytest_workflow/plugin.py

```diff
@@ -185,28 +261,34 @@ def queue_workflow(self):
         This is shorter than using pytest's terminal reporter.
         """
 
-        self.tempdir = (self.config.workflow_dir /
-                        Path(replace_whitespace(self.name, '_')))
+        tempdir = (self.config.workflow_temp_dir /
```

Comment:
This makes the API backwards incompatible - which would mean version 2.0.0 per semver. 

Is there anything against keeping `tempdir` as an instance field, and just not doing to teardown here anymore?

Reply:
The API is the YAML file in this case. And also the tests files that users write.
This does not interfere with it. There is no user-side documentation on how to use the internals of the code.
I prefer to keep as little instance variables as possible and this one just is not necessary. It was only there for the teardown method.